Defs/CE_FlareTest.xml

```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
+  <!-- ==================== Flare Motes ========================== -->
+
+  <ThingDef ParentName="MoteBase">
+    <defName>Mote_FlareGlow</defName>
+    <thingClass>CombatExtended.MoteThrownCE</thingClass>
+    <graphicData>
+      <texPath>Things/Mote/FireGlow</texPath>
+      <shaderType>MoteGlow</shaderType>
+    </graphicData>
+    <altitudeLayer>MoteOverhead</altitudeLayer>
+    <mote>
+      <fadeInTime>0.75</fadeInTime>
+      <solidTime>1.08</solidTime>
+      <fadeOutTime>0.8</fadeOutTime>
+    </mote>
+  </ThingDef>
+
+  <ThingDef ParentName="MoteBase">
+    <defName>Mote_FlareSmoke</defName>
+    <thingClass>CombatExtended.MoteThrownCE</thingClass>
+    <graphicData>
+      <texPath>Things/Mote/Smoke</texPath>
+    </graphicData>
+    <altitudeLayer>MoteOverhead</altitudeLayer>
+    <mote>
+      <fadeInTime>0.50</fadeInTime>
+      <solidTime>6</solidTime>
+      <fadeOutTime>3.2</fadeOutTime>
+      <growthRate>0.005</growthRate>
+    </mote>
+  </ThingDef>
+
+  <!-- ==================== Flare ThingCategoryDef ========================== -->
+
+  <ThingCategoryDef>
+    <defName>AmmoFlare</defName>
+    <label>flare</label>
+    <parent>AmmoShotguns</parent>
+    <iconPath>UI/Icons/ThingCategories/CaliberShotgun</iconPath>
+  </ThingCategoryDef>
+
+  <!-- ==================== AmmoSet ========================== -->
+
+  <CombatExtended.AmmoSetDef>
+    <defName>AmmoSet_Flare</defName>
+    <label>12 Gauge</label>
```

Comment:
Probably want a different label

Reply:
The Xml is only a placeholder since i don't do Xml it's more of a guideline for the team...